### PR TITLE
NUI-4708 docs for two directional stacked area chart

### PR DESCRIPTION
**Pull request by Blakesalot**

### CODE REVIEW

File: packages/charts/.gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ sdk/
 dist/
 **/coverage/
 .tmp
+_snapshots
```
Comment (madelyneh) [inline packages/charts/.gitignore:6]: Just curious, but what are they snapshots?

Comment (Blakesalot) [reply packages/charts/.gitignore]: This is from before the merge from main which already has this change. Lukas added this in his PR. It's the directory that gets created when the Percy visual test build task is run.

File: packages/charts/examples/components/demo/chart-types/area/chart-docs-area.module.ts [hunk 2]
```diff
@@ -18,6 +20,7 @@ import { AreaChartVariableBaselineExampleComponent } from "./area-chart-variable
 import { AreaChartVerticalExampleComponent } from "./area-chart-vertical/area-chart-vertical-example.component";
 import { ChartDocsAreaComponent } from "./chart-docs-area.component";
 import { AreaChartTestComponent } from "./test/area-chart-test.component";
+/* eslint-enable max-len */
```
Comment (madelyneh) [inline packages/charts/examples/components/demo/chart-types/area/chart-docs-area.module.ts:23]: Isn't this on globally? why is it needed here?

**Review by madelyneh — APPROVED** [review]

**Review by baggiosw — APPROVED** [review]

File: packages/charts/examples/components/demo/chart-types/area/chart-docs-area.module.ts (hunk 2 above)
Comment (Blakesalot) [reply packages/charts/examples/components/demo/chart-types/area/chart-docs-area.module.ts:23]: I actually disable it at the top of the file. This is just re-enabling it afterwards.

**Review by madelyneh — APPROVED** [review]

**Review by lukasholcik — APPROVED** [review]

**Pull request merged**